index.html

```diff
@@ -28,6 +28,19 @@ <h1>Trouvez votre hébergement pour des vacances de rêve</h1>
                 <input type="text" id="recherche" name="recherche_par_ville" placeholder="Marseille, France">
                 <input type="submit" value="Rechercher">
             </form>
+            <div id="container-filter">
```

Comment:
Better to close section and reopen a new. Have a first section with id="search" and a a second with id "filter"